examples/gno.land/p/demo/forms/create.gno

```diff
@@ -0,0 +1,95 @@
+package forms
+
+import (
+	"std"
+	"time"
+
+	"gno.land/p/demo/json"
+	"gno.land/p/demo/ufmt"
+)
+
+func CreateField(label string, fieldType string, required bool) Field {
```

Comment:
In relation to code organization; 

From what I can see, you wanted a single file to handle all "creation" of specific structs you use in your package. 

If you are splitting by file, I would separate the functionality by struct, and not by functionality. You shouldp probably put this in the `form.gno` file which would handle all form-related stuff, same for others. 

This might be a nitpick; I am open to others giving their opinion on this.

Reply:
Looking at this again, why is `ParseDates`:
1. In this file
2. Public

If you're using it only internally, it should be private (unxeported)

Reply:
The idea was to separate the code between "creating a form" and "submitting a form" 
> (I also put validation in another file because it contains a lot of code, but it could/should go in `submit.gno` probably)

So it's not really about the 'creation' functions, but really about the 'form creation process'
Which is why `ParseDates` is there since it's used only when creating a new form

But you're probably right that the organisation isn't optimal, so we'll see what others think about it

Reply:
made `ParseDates` -> `parseDates` (unexported) in https://github.com/gnolang/gno/pull/2604/commits/76afc4662e7e2e1097c6f0492c1b4834b7dc61ae